doc/command_ref.rst

```diff
@@ -1109,6 +1109,8 @@ packages to those matching the specification. All packages are considered if no
     in the configuration file might influence the result, but if the command line option  \-\
     :ref:`-disableexcludes <disableexcludes-label>` is used, it ensures that all installed packages will be listed.
 
+.. _whatdepends_optin-label:
```

Comment:
You're missing an 'o' in "option", should be: `_whatdepends_option-label`.

Reply:
Fixed